types/alexa-sdk/tslint.json

```diff
@@ -4,6 +4,7 @@
       "object-literal-key-quote": false,
       "no-empty-interface": false,
       "prefer-method-signature": false,
-      "object-literal-key-quotes": false
+      "object-literal-key-quotes": false,
+      "no-any": false
```

Comment:
Can you use the `//tslint:disable-next-line rule-name` option to disable this check just for the locations that are affected instead of disabling for the whole definition. This way new issues are less likely to creep in.